metadata/root.json

```diff
@@ -2,17 +2,17 @@
  "signatures": [
   {
    "keyid": "c8e09a68b5821b75462ae0df52151c81deb7f1838246dc1da8c34cc91ec12bda",
-   "sig": "3045022016f788e5c90b169b4a9b28c56bf73df988b67c2d73f4e49205db50f9f0ae8066022100b4aabd9298ff12205724bae48bf9118a262581a7ca4b5d1a1b4c68498bd5fdde"
+   "sig": ""
```

Comment:
This is https://github.com/theupdateframework/tuf-on-ci/issues/157, it's not harmful and will disappear when the next metadata version is made